maquetado/setup/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+$tech-logo-size: 100px;
+$tech-container-margin: 50px;
+$tech-container-height: 300px;
+$tech-container-width: 210px;
+
+h2 {
```

Comment:
Try to avoid using html elements for tags, its always better to create css classes